### ISSUE TITLE
使用docker转换pb模型失败

### ISSUE THREAD

**Issue by fire717** [Tencent/TNN#161]
参考convert2tnn说明，到最后一步执行
`docker run --volume=$(pwd):/workspace -it tnn-convert:latest  python3 ./converter.py tf2tnn -tp /workspace/224_1.0_epoch1_1.0.pb -in "input_1[1,224,224,3]" -on output_1 -v v1.0 -optimize -align
`

报错：
`
----------  convert model, please wait a moment ----------

Converter Tensorflow to TNN model

Convert TensorFlow to ONNX model succeed!

Converter ONNX to TNN Model

const node = node_bn_Conv1/cond/Merge:0
const node = node_Conv1_relu/Relu6:0
const node = node_expanded_conv_depthwise/depthwise__796:0
const node = node_expanded_conv_depthwise/depthwise:0
const node = node_expanded_conv_depthwise_BN/FusedBatchNormV3:0
...
const node = node_global_average_pooling2d_1/Mean:0
const node = node_dense_1/MatMul:0
const node = node_dense_1/BiasAdd:0
const node = node_dense_1/Softmax:0
const node = node_strided_slice:0
const node = node_output_1:0
E/converter: int Onnx2TNN::TNNWriteProto() [File /opt/TNN/tools/onnx2tnn/src/core/onnx2tnn.cc][Line 290] invalid output blob count(must = 1): 0
python3: /opt/TNN/tools/onnx2tnn/src/core/onnx2tnn.cc:291: int Onnx2TNN::TNNWriteProto(): Assertion `0' failed.
Aborted (core dumped)

Converter ONNX to TNN model failed!
`

模型是mobilenet v2，用的keras训练然后保存h5模型转为pb，已验证pb模型输出和h5一致。
[224_1.0_epoch1_1.0.zip](https://github.com/Tencent/TNN/files/4888311/224_1.0_epoch1_1.0.zip)

**Comment by fire717** [comment]
尝试分步处理，
pb转onnx成功，且测试onnx输出和pb一致。
然后onnx转tnn失败，报错和上述一致。

**Comment by 1627180283** [comment]
你转换出来的 onnx 模型能给我看一下吗？我这边测试转换出来之后的 onnx 模型有问题

**Comment by fire717** [comment]
> 你转换出来的 onnx 模型能给我看一下吗？我这边测试转换出来之后的 onnx 模型有问题

你好，经排查，是h5转pb时未设置为非训练，导致多了一个输入节点。
重新生成pb后，尝试分步处理转onnx再转tnn，第二步会有这个issue一样的报错：
https://github.com/Tencent/TNN/issues/110

但是直接用docker命令一键转换成功了。

修改后的onnx文件如下你们可以进行排查那个issue提到的问题。
[test_new.zip](https://github.com/Tencent/TNN/files/4888964/test_new.zip)
